app/views/declaration/office_of_exit.scala.html

```diff
@@ -30,11 +30,13 @@
 
         @components.error_summary(form.errors)
 
+        @components.section_header(messages("supplementary.summary.locations.header"))
+
         @components.page_title(Some("supplementary.officeOfExit.title"))
 
         @components.input_text(
             field = form("officeId"),
-            label = messages("supplementary.officeOfExit"),
+            label = "",
```

Comment:
Just to be sure. Do you want to remove label?

Reply:
Yes. We have a hint, but no longer a label